src/qt3utils/applications/qt3ple/ple_scan_loader.ipynb

```diff
@@ -0,0 +1,265 @@
w+H7Zt2wYAKCwshN1u13ZwzTB79mxMmzYNgwcPlrJ6hw8fbjO1XzU1Ndi3b5/0+cGDB7Ft2zakpaWhc+fOGo5MPffccw8WLVqEzz77DA6HQ8r4OZ1OWK1WjUfXco899hiuuOIK5OXlobq6GosXL8Y333yDZcuWaT001TgcjpCaL1Zr2lZqwR588EFMmjQJnTt3RllZGX73u9+hqqoKt9xyS+sMQLP1bSTEDz/8wI8YMYJPS0vjzWYzn5+fz99555380aNHtR6aKhYsWMADCPuvLbnlllvCPsavv/5a66E12/z58/kuXbrwJpOJHzhwYJtarv3111+H/XvdcsstWg9NNZFedwsWLNB6aKr41a9+JT0/O3TowI8cOZL/6quvtB5W3LW1ZffXX38937FjR95oNPI5OTn8Nddcw+/cubPVfj7H8wlcrUsIIYQQ0grOjQIOQgghhJA4ooCIEEIIIe0eBUSEEEIIafcoICKEEEJIu0cBESGEEELaPQqICCGEENLuUUBECCGEkHaPAiJCCCGEtHsUEBFCzjn5+fl47bXXtB5Go/76179izJgxMX1PWVkZOnTogGPHjsVpVISQSCggIoS0mkmTJmHUqFFhv7Z+/XpwHIfvvvsu5vvlOA6ffvppC0enHpfLhSeffBJPPPEEAGGX+e7du4c99tixY9Dr9fjkk0+QmZmJadOm4amnnmrN4RJCQAERIaQVTZ8+HatWrcLPP/8c8rV3330X559/PgYOHKjByNT18ccfw263S7t0T58+Hfv27cPatWtDjl24cCHS09MxadIkAMIO5u+//z7Ky8tbdcyEtHcUEBFCWs3EiRORmZmJhQsXKm6vq6vDBx98gOnTpwMQAorevXvDbDYjPz8ff/jDHyLeZ35+PgDg6quvBsdx0uf79+/HlVdeiaysLNjtdlxwwQVYsWKF4ntPnDiBCRMmwGq1oqCgAIsWLQqZjqusrMSMGTOQmZmJ5ORkXH755fj+++8bfZyLFy/G5MmTpc9ZoPfuu++GHLtw4ULcfPPNMBqNAIC+ffsiOzsbS5YsafRnEELURQERIaTVGAwG3HzzzVi4cCHk+0p/+OGHcLvduOmmm7BlyxZMmTIFN9xwA7Zv346nn34aTzzxREgQxWzatAkAsGDBApw4cUL6vKamBuPHj8eKFSuwdetWjB07FpMmTcLhw4el77355ptx/PhxfPPNN/j444/xzjvvoKysTPo6z/OYMGECSktL8cUXX2DLli0YOHAgRo4cibNnz0Z8nGvXrsXgwYMVt02fPh0ffvghampqpNtWr16Nffv24Ve/+pXi2CFDhoTNJhFC4ognhJBWtGvXLh4Av2rVKum2Sy+9lL/xxht5nuf5qVOn8qNHj1Z8z0MPPcQXFRVJn3fp0oV/9dVXpc8B8EuWLGnyZxcVFfGvv/66YhybNm2Svr53714egHTfK1eu5JOTk/mGhgbF/XTr1o1/++23w/6M8vJyHgC/Zs2akNstFgv/7rvvSrfdfPPNfHFxcch9/OY3v+GHDx/e5OMhhKiHMkSEkFbVs2dPDBs2TJo+2r9/P9auXStlSXbt2oWLLrpI8T0XXXQR9u7dC5/PF/XPqa2txcMPP4yioiKkpKTAbrdj9+7dUoZoz549MBgMipqlwsJCpKamSp9v2bIFNTU1SE9Ph91ul/4dPHgQ+/fvD/tz6+vrAQAWi0Vxe0pKCq655hrpcVdXV+Pjjz8OyQ4BgNVqRV1dXdSPlRDScgatB0AIaX+mT5+Oe++9F/Pnz8eCBQvQpUsXjBw5EoAwTcVxnOJ4Xja9Fq2HHnoIX375JX7/+9+jsLAQVqsV1113Hdxud6P3Kb/d7/ejY8eO+Oabb0KOS0lJCfv96enp4DgubFH09OnTMXLkSOzduxerV68GAFx//fUhx509exYdOnRo6iESQlREGSJCSKubMmUK9Ho9Fi1ahPfeew+33XabFAQVFRVh3bp1iuNLSkpw3nnnQa/Xh70/o9EYkj1au3Ytbr31Vlx99dVSofKhQ4ekr/fs2RNerxdbt26Vbtu3bx8qKiqkzwcOHIjS0lIYDAYUFhYq/mVkZIQdi8lkQlFREX788ceQr40YMQJdu3bFwoUL8e6772LKlClwOBwhx+3YsQMDBgwIe/+EkPiggIgQ0ursdjuuv/56PPbYYzh+/DhuvfVW6WsPPPAAVq5cieeeew4//fQT3nvvPcybNw8PPvhgxPvLz8/HypUrUVpaKmVmCgsL8cknn2Dbtm34/vvvMXXqVPj9ful7evbsiVGjRmHGjBnYuHEjtm7dihkzZsBqtUrB2ahRo1BcXIyrrroKX375JQ4dOoSSkhL83//9HzZv3hxxPGPHjg0J6gChX9Jtt92GN998E+vXr5dW1cnV1dVhy5YtMTd1JIS0kLYlTISQ9qqkpIQHwI8ZMybkax999BFfVFTEG41GvnPnzvzLL7+s+HpwUfXnn3/OFxYW8gaDge/SpQvP8zx/8OBBfsSIEbzVauXz8vL4efPm8Zdddhl///33S993/Phx/oorruDNZjPfpUsXftGiRXxmZib/1ltvScdUVVXxM2fO5HNycnij0cjn5eXxN910E3/48OGIj23Xrl281WrlKyoqQr525MgRXqfT8T169Aj7vYsWLYr4NUJI/HA834zJeUIIaYOOHj2KvLw8rFixQqppaq4pU6ZgwIABePTRR2P6viFDhmDWrFmYOnVqi34+ISQ2NGVGCGm3Vq1ahc8//xwHDx5ESUkJbrjhBuTn5+PSSy9t8X2//PLLsNvtMX1PWVkZrrvuOtx4440t/vmEkNhQhogQ0m59+eWXeOCBB3DgwAE4HA4MGzYMr732Grp06aL10AghrYwCIkIIIYS0ezRlRgghhJB2jwIiQgghhLR7FBARQgghpN2jgIgQQggh7R4FRIQQQghp9yggIoQQQki7RwERIYQQQto9CogIIYQQ0u79PxmAKa8HkydBAAAAAElFTkSuQmCC",
```

Comment:
What is this lol...

Reply:
Apparently this is what happens to local image files in jupyter notebook. I think it can be ignored since the actual notebook renders correctly.